tuplex/core/src/HistoryServerConnector.cc

```diff
@@ -147,6 +151,35 @@ namespace tuplex {
         json obj;
         obj["job"] = job;
 
+        // add operators...
+        std::vector<json> ops;
+        //        TransformStage* trafoStage = dynamic_cast<TransformStage*>(_stage);
+        assert(plan);
+        plan->foreachStage([&](const PhysicalStage* stage) {
+            for(auto op: stage->get_ops()) {
+                json val;
+                val["name"] = op->name();
+                val["id"] = "op" + std::to_string(op->getID());
+                // @Todo: solve this...
```

Comment:
what is this TODO about? Can we solve it?

Reply:
This is an old leftover TODO. It is now deleted.